### PR TITLE
Add new ServiceNow London API type definitions

### PR DESCRIPTION
**Pull request by johncaruso**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/servicenow-london/tslint.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (sheetalkamat) [inline types/servicenow-london/tslint.json]: Please remove everything from here except extends clause. New packages cannot have custom tslint rules/options

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/servicenow-london/tslint.json (hunk 1 above)
Comment (johncaruso) [reply types/servicenow-london/tslint.json]: OK. Done.

**Comment by johncaruso** [comment]
Will the bot update this to get another review?

**Comment by johncaruso** [comment]
@sheetalkamat I've made changes, all checks have passed. Shouldn't this get tagged with Awaiting reviewer feedback? Just want to make sure you're not waiting on me for something further.

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**